src/sync/incomingSyncUtils.js

```diff
@@ -725,6 +725,8 @@ export const createOrUpdateRecord = (database, settings, recordType, record) =>
         enteredBy,
         otherParty,
         linkedRequisition,
+        option: database.getOrCreate('Options', record.optionID),
+        total: parseFloat(record.total),
```

Comment:
These fields are already added to the `Transaction` object, but looks like the incoming sync logic wasn't added.

Reply:
yeah - just added when starting dispensing but hadn't done a lot of the sync stuff.

The `total` was added - at the moment we derive the total from all of the trans lines of the transaction quantity * price - that's gonna need to change soon as the payment trans lines won't have a quantity :(